main.js

```diff
@@ -0,0 +1,106 @@
+function grab(e, isId = false, qAll = false) {
+  if (isId) {
+    return document.getElementById(e);
+  } if (qAll) {
+    return document.querySelectorAll(`.${e}`);
+  }
+  return document.querySelector(`.${e}`);
+}
```

Comment:
- [ ] Kindly change this to es6 arrow function syntax also.

Reply:
fixed